src/lua/mod.rs

```diff
@@ -124,14 +129,17 @@ pub fn init() {
 fn thread_init(sender: Sender<LuaResponse>, receiver: Receiver<LuaQuery>) {
     trace!("thread: initializing.");
     let mut lua = Lua::new();
+    //unsafe {
+    //    hlua_ffi::lua_atpanic(&mut lua.as_mut_lua().0, thread_on_panic);
+    //}
```

Comment:
Ah, is this not previous code you commented out? I see, if it was I was going to say you could just use version control and revert the commit that removed it, but I see now that you are adding this as a comment. Ok, that's fine then.